app/views/katello/api/v2/products/show.json.rabl

```diff
@@ -3,16 +3,18 @@ object @resource
 
 extends "katello/api/v2/products/base"
 
-extends 'katello/api/v2/common/org_reference'
-
 attributes :productContent => :product_content
 
 attributes :available_content => :available_content
 
+attributes :redhat? => :redhat
+
 child :library_repositories => :repositories do |_repo|
   attributes :name, :id
 end
 
+attributes :available_content => :available_content, :if => params[:include_available_content]
```

Comment:
Ah okay, thanks for pointing that out. I"ll add this back into ``base.json.rabl``.